src/scripts/main.js

```diff
@@ -1,7 +1,30 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const div = document.createElement('div');
```

Comment:
```suggestion
  const notification = document.createElement('div');
```
Don't name variables the same as tag. Try to give them more descriptive names